pkg/apis/v2beta2/receiver_types.go

```diff
@@ -201,6 +201,17 @@ type SmsReceiver struct {
 	Template *string `json:"template,omitempty"`
 }
 
+type PushoverReceiver struct {
+	// whether the receiver is enabled
+	Enabled *bool `json:"enabled,omitempty"`
+	// PushoverConfig to be selected for this receiver
+	PushoverConfigSelector *metav1.LabelSelector `json:"pushoverConfigSelector,omitempty"`
+	// Selector to filter alerts.
+	AlertSelector *metav1.LabelSelector `json:"alertSelector,omitempty"`
+	// The users (Pushover User Keys) to send notifications to.
+	UserKeys []string `json:"userKeys"`
```

Comment:
The receiver needs to support to set template like [this](https://github.com/kubesphere/notification-manager/blob/master/pkg/apis/v2beta2/receiver_types.go#L95).

Reply:
Resolved: https://github.com/txfs19260817/notification-manager/blob/0e75498c02eb381666cbb63da674b9714b0bfb74/pkg/apis/v2beta2/receiver_types.go#L215